slave.mk

```diff
@@ -1023,36 +1033,40 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 		chmod +x sonic_debian_extension.sh,
 	)
 
-	DEBUG_IMG="$(INSTALL_DEBUG_TOOLS)" \
-	DEBUG_SRC_ARCHIVE_DIRS="$(DBG_SRC_ARCHIVE)" \
-	DEBUG_SRC_ARCHIVE_FILE="$(DBG_SRC_ARCHIVE_FILE)" \
-		scripts/dbg_files.sh
-
-	DEBUG_IMG="$(INSTALL_DEBUG_TOOLS)" \
-	DEBUG_SRC_ARCHIVE_FILE="$(DBG_SRC_ARCHIVE_FILE)" \
-	USERNAME="$(USERNAME)" \
-	PASSWORD="$(PASSWORD)" \
-	IMAGE_TYPE=$($*_IMAGE_TYPE) \
-	TARGET_PATH=$(TARGET_PATH) \
-	SONIC_ENFORCE_VERSIONS=$(SONIC_ENFORCE_VERSIONS) \
-	TRUSTED_GPG_URLS=$(TRUSTED_GPG_URLS) \
-	PACKAGE_URL_PREFIX=$(PACKAGE_URL_PREFIX) \
-	MULTIARCH_QEMU_ENVIRON=$(MULTIARCH_QEMU_ENVIRON) \
-		./build_debian.sh $(LOG)
-
-	USERNAME="$(USERNAME)" \
-	PASSWORD="$(PASSWORD)" \
-	TARGET_MACHINE=$($*_MACHINE) \
-	IMAGE_TYPE=$($*_IMAGE_TYPE) \
-	SONIC_ENABLE_IMAGE_SIGNATURE="$(SONIC_ENABLE_IMAGE_SIGNATURE)" \
-	SIGNING_KEY="$(SIGNING_KEY)" \
-	SIGNING_CERT="$(SIGNING_CERT)" \
-	CA_CERT="$(CA_CERT)" \
-	TARGET_PATH="$(TARGET_PATH)" \
-		./build_image.sh $(LOG)
+	# Build images for the MACHINE, DEPENDENT_MACHINE defined.
+	$(foreach dep_machine, $($*_MACHINE) $($*_DEPENDENT_MACHINE), \
```

Comment:
We can improve it to run the two images in parallel, it increases about 20 minutes in PR build.
2021-06-19T06:33:00.3974964Z [ building ] [ target/sonic-broadcom.bin ] 
2021-06-19T07:29:38.9766316Z [ finished ] [ target/sonic-broadcom.bin ]
But it is not a block issue, we can improve it later.

Reply:
Sure thanks @xumia will raise this as a follow on PR -- to build both images in parallel.